lib/core/sdam/topology.js

```diff
@@ -110,6 +117,14 @@ class Topology extends EventEmitter {
     }
 
     options = Object.assign({}, common.TOPOLOGY_DEFAULTS, options);
+    DEPRECATED_OPTIONS.forEach(optionName => {
+      if (options[optionName]) {
+        process.emitWarning(
```

Comment:
`process.emitWarning` is not supported in node 4. You should probably export [this function](https://github.com/mongodb/node-mongodb-native/blob/master/lib/utils.js#L581) and use that instead.

Reply:
done